packages/react/src/Card/Card.module.css

```diff
@@ -78,7 +86,7 @@
   align-self: center;
 }
 
-.Card--colorMode-light:not(.Card--disableAnimation):hover {
+.Card:not(.Card--disableAnimation, .Card--variant-default.Card--colorMode-dark):hover {
   transform: scale3d(1.025, 1.025, 1);
```

Comment:
@rezrah can we use a more subtle opacity fade instead?

It's looking very bouncy, too playful. The easing is fine but I'd suggest removing the `scale3d`

Reply:
This is the current default for cards, so making this change would change all light mode cards globally. Is that something we can defer as we'd need more time to investigate impact.